DependencyInjection/MonologExtension.php

```diff
@@ -110,6 +110,16 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         $container->setParameter('monolog.additional_channels', isset($config['channels']) ? $config['channels'] : array());
+
+        if ($config['process_messages_using_psr_3_rules']) {
+            $processor = new Definition("Monolog\\Processor\\PsrLogMessageProcessor");
+            $processorId = uniqid('monolog.psr_log_message.processor.');
```

Comment:
Why create one processor per handler that needs it? I think we could just have one fixed service id, as it is stateless  and configuration-less, a unique service can be injected in all handlers requiring it.